ISISComputingGroup/ibex_gui
# Ticket1063 web browser for weblinks

ISISComputingGroup/IBEX/issues/1063

To test:
* Check new Web Links perspective behaves in the same way as the old one

Might be good to check this on Linux too.
